arch/armv7a/jmp.S

```diff
@@ -14,29 +14,28 @@
  * %LICENSE%
  */
 
- /* setjmp env buffer description
-  * Offset  Description
-  * 0       Signal mask flag - if != 0 then next value stores a signal mask
-  * 4       Signal mask
-  * 8       SP register
-  * 12-43   r4-r11 registers
-  * 44      LR register
-  * 48-79   d8-d15 registers
-  * 80      fpscr register
-  */
+/* setjmp env buffer description
+ * Offset  Description
+ * 0       Signal mask flag - if != 0 then next value stores a signal mask
+ * 4       Signal mask
+ * 8       SP register
+ * 12-43   r4-r11 registers
+ * 44      LR register
+ * 48-103  d8-d15 registers
```

Comment:
Hmm, I think it was correct before:
- 48 d8
- 52 d9
- 56 d10
- 60 d11
- 64 d12
- 68 d13
- 72 d14
- 76 d15
- 80 fpscr